augly/image/utils/bboxes.py

```diff
@@ -444,6 +445,34 @@ def get_enclosing_bbox(
     )
 
 
+def spatial_bbox_helper(
+    bbox: Tuple, src_w: int, src_h: int, aug_function: Callable, **kwargs
+) -> Tuple:
+    """
+    Computes the bbox that encloses the transformed bbox in the image transformed by
+    `aug_function`. This helper can be used to compute the transformed bbox for any
+    augmentation which doesn't affect the color of the source image (e.g. any spatial
+    augmentation).
+    """
+    dummy_image = Image.new("RGB", (src_w, src_h))
+    draw = ImageDraw.Draw(dummy_image)
+    draw.rectangle(
+        [bbox[0] * src_w, bbox[1] * src_h, bbox[2] * src_w, bbox[3] * src_h],
```

Comment:
our internal linter errored with:
```
Incompatible parameter type [6]: Expected typing.Union[Tuple[Tuple[float, float], Tuple[float, float]], Tuple[float, float, float, float]] for 1st positional only parameter to call ImageDraw.ImageDraw.rectangle but got List[typing.Any].
```

Here's a potential way to fix this:
```suggestion
    bbox: Tuple[float, float, float, float],
    src_w: int,
    src_h: int,
    aug_function: Callable,
    **kwargs,
) -> Tuple:
    """
    Computes the bbox that encloses the transformed bbox in the image transformed by
    `aug_function`. This helper can be used to compute the transformed bbox for any
    augmentation which doesn't affect the color of the source image (e.g. any spatial
    augmentation).
    """
    dummy_image = Image.new("RGB", (src_w, src_h))
    draw = ImageDraw.Draw(dummy_image)
    draw.rectangle(
        (bbox[0] * src_w, bbox[1] * src_h, bbox[2] * src_w, bbox[3] * src_h),
```

This *should* help ensure that what's being passed into `draw.rectange` is in fact a `Tuple[float, float, float, float]`